ga4gh/backend.py

```diff
@@ -708,3 +729,90 @@ def __init__(self, dataDir):
             dataset = datasets.FileSystemDataset(datasetDir)
             self._datasetIdMap[dataset.getId()] = dataset
         self._datasetIds = sorted(self._datasetIdMap.keys())
+
+
+class GeneotypePhenotypeBackend(AbstractBackend):
+    """
+    an rdf object store
+    """
+    # keep the graph as a static variable
+    def __init__(self, is_prod):
+        super(GeneotypePhenotypeBackend, self).__init__()
+        self.dataset = genotype_phenotype.G2PDataset(is_prod)
+
+
+class GeneotypePhenotypeIterator(IntervalIterator):
```

Comment:
I don't think we need this class. The IntervalIterator infrastructure is needed for reads and variants as they are both defined over genomic intervals, and we can only query using start and end coordinates. We therefore have to do complicated things to make sure we don't return the same result several times. For G2P associations, we just need an offset into the query, right?